source/Calamari.Tests/Fixtures/Bash/Scripts/report-kubernetes-manifest.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+manifest="""
+\"apiVersion\": \"v1\"
+\"kind\": \"Namespace\"
+\"metadata\":
+  \"name\": \"example\"
+\"labels\":
+    \"name\": \"example\"
+---    
+\"apiVersion\": \"v1\"
+\"kind\": \"Namespace\"
+\"metadata\":
+  \"name\": \"diffs\"
+\"labels\":
+    \"name\": \"diffs\"
+"""
```

Comment:
Using Heredoc can clean up those escapes (disclaimer, I didn't test this)

```
read -r -d '' manifest << EOM
"apiVersion": "v1"
"kind": "Namespace"
"metadata":
  "name": "example"
"labels":
    "name": "example"
---    
"apiVersion": "v1"
"kind": "Namespace"
"metadata":
  "name": "diffs"
"labels":
    "name": "diffs"
EOM
```

Reply:
I might leave it as it's only a test script